content/blog/2021-02-10-onboarding-resource-roundup/contents.lr

```diff
@@ -0,0 +1,67 @@
+
+- [Development Setup](https://inveniordm.docs.cern.ch/develop/): Install a development version
+
+- [Development Workflows](https://inveniordm.docs.cern.ch/maintenance/development_workflow/): Learn about the development workflows used
+
+### Community Members
+
+All stakeholders at your institution are welcome to contribute to InvenioRDM, whether their contribution is code-based or not. To create a tool to best serve the community, outlining stakeholder requirements will be vital:
+
+ - [Demo site](https://inveniordm.web.cern.ch/): Access the latest sandbox version of InvenioRDM here. Create an account to test record creation
+
+ - [Discourse](https://invenio-talk.web.cern.ch/): Provide feedback on testing here. Use the [Feedback](https://invenio-talk.web.cern.ch/c/site-feedback/2) category or start a new thread wherever you like
+
+ - [Metadata Interest Group](https://invenio-talk.web.cern.ch/t/metadata-interest-group/144): This is the first community-focused group organized to help support the InvenioRDM development effort. This group meets about once/month and discusses issues related to the [InvenioRDM Metadata Model](https://inveniordm.docs.cern.ch/reference/metadata/). To join, either make a post on the [Discourse thread](https://invenio-talk.web.cern.ch/t/metadata-interest-group/144) or in this dedicated [Discord channel](https://discord.gg/hqvDW9p7Cx)
+
+ - [Conference Calendar](https://docs.google.com/document/d/1HhGtd39TqxleUkK-b5qyPqMjSg8EBqacxSpoDtydmE8/edit?usp=sharing): Are you presenting on InvenioRDM at an upcoming conference? Make a note here so other interested members of the community can attend the session, or perhaps join your poster or presentation effort
+
+ - Welcome, Help and General Questions: Reach out to Community Manager Sara Gonzales via [Discord](https://discordapp.com/users/saragon7740) for general questions or to join the Metadata Interest Group
+
+ - Schedule a Demo: Is your site, or a site you know, considering implementing InvenioRDM? Reach out to Community Manager Sara Gonzales via [Discord](https://discordapp.com/users/saragon7740) to schedule a one-hour demo and information session about the repository
+ 
+
+ ### Everyone
+
+ - Project list-serv: Lars will have shared this email address with you upon joining the team. This is the frontline communication tool for the InvenioRDM community. Make this email address a safe sender to receive the latest updates on Telecons and community-based information such as invitations to the Metadata Interest Group meetings. Polls, forms, and additional community tools will be sent to this list-serv
+
+ - Telecons (see the list-serv for connection information): All partners of the InvenioRDM community are invited to bi-weekly Telecons throughout the year. One Telecon per month will focus on development updates, and the other will focus on community-based topics
```

Comment:
Telecon -> telecon (is there a reason for the casing?)

Reply:
Updated to lower case